doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1233,6 +1160,58 @@ Other API Changes
 - :func:`Series.to_list` and :func:`Index.to_list` are now aliases of ``Series.tolist`` respectively ``Index.tolist`` (:issue:`8826`)
 - The result of ``SparseSeries.unstack`` is now a :class:`DataFrame` with sparse values, rather than a :class:`SparseDataFrame` (:issue:`24372`).
 
+
+.. _whatsnew_0240.api.extension:
+
+ExtensionType Changes
+^^^^^^^^^^^^^^^^^^^^^
+
+**Equality and Hashability**
+
+Pandas now requires that extension dtypes be hashable. The base class implements
```

Comment:
Seems I'm a bit too late to mention this on time. In https://github.com/pandas-dev/pandas/issues/22476#issuecomment-440349264, we talked about this line.

How about: Pandas now requires that extension dtypes be hashable *(i.e. the respective `ExtensionDtype`-objects; hashability is not a requirement for the values of the corresponding `ExtensionArray`)*.